docs/anomaly-detection/docs/models/models.md

```diff
@@ -0,0 +1,325 @@
+---
+sort: 1.1
+weight: 1.1
+title: Built-in Models
+# disableToc: true
+menu:
+  docs:
+    parent: "models"
+    sort: 1.1
+    # weight: 1.1
+aliases:
+  - /anomaly-detection/docs/models/models.html
+---
+
+# Models config parameters
+
+## Section Overview
+VM Anomaly Detection (`vmanomaly` hereinafter) models support 2 groups of parameters:
+
+- **`vmanomaly`-specific** arguments 
+  - To run wrapper (say, for `model.prophet.ProphetModel`). Defined in *Parameters specific for vmanomaly* sections for each model below.
```

Comment:
I don't think the user needs to know why we use specific params. This is about our inner architecture and doesn't affect vmanomaly usage.
These explanations may confuse. I suggest skipping them.

Reply:
as a tradeoff, changed to 

- **`vmanomaly`-specific** arguments - please refer to *Parameters specific for vmanomaly* and *Default model parameters* subsections for each of the models below.
- Arguments to **inner model** (say, [Facebook's Prophet](https://facebook.github.io/prophet/docs/quick_start.html#python-api)), passed in a `args` argument as key-value pairs, that will be directly given to the model during initialization to allow granular control. Optional.

so the mention of param subsections is present but the (internal) intention is gone